src/components/info_pvr.coffee

```
@@ -63,6 +63,7 @@ InfoPvr = React.createClass
       headerClass: ''
       pvrProps: {}
       className: 'plain-pvr'
+      close: ->
```

Looks like the prop `close` is set to `isRequired` in propTypes. We shouldn't follow this pattern because it essentially never fails. If the user doesn't pass a `close` method, then it will be covered by `getDefaultProps`.

Instead, we should remove this line, and let the component throw a propTypes error if the user doesn't pass it.